src/PrestaShopBundle/Controller/Admin/PrestaShopAdminController.php

```diff
@@ -229,6 +230,57 @@ protected function getErrorMessageForException(Throwable $e, array $messages): s
         );
     }
 
+    /**
+     * Returns form errors for JS implementation.
+     *
+     * Parse all errors mapped by id html field
+     *
+     * @param FormInterface $form
+     *
+     * @return array<array<string>> Errors
+     *
+     * @throws \Symfony\Component\Translation\Exception\InvalidArgumentException
+     */
+    protected function getFormErrorsForJS(FormInterface $form): array
```

Comment:
This looks like something that could be unit tested

Reply:
I agree, but in another PR could be better 🤔

Reply:
Also moved out, because private and protected methods should not be tested.